src/scenes/gameplay-scene/gameplay-scene.ts

```
@@ -706,379 +477,75 @@ export class GameplayScene {
-      this.tutorial = null;
-    }
-
-    // Clear event listeners
-
-    if (this.unsubscribeEvent) {
-      this.unsubscribeEvent();
-      this.unsubscribeMiniGameEvent();
-      this.unsubscribeLoadGamePuzzle();
-      this.unsubscribeEvent = null;
+    if (this.eventListenersAdded) {
+      this.eventListenersAdded.forEach(unsubscribe => unsubscribe());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)